webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/RegistrationCommandsPageSelectors.js

```diff
@@ -9,11 +9,24 @@ import {
   selectAPIResponse,
 } from '../../../redux/API/APISelectors';
 
+import { STATUS } from '../../../constants';
+
 // Form API Data
 
 export const selectAPIStatusData = state =>
   selectAPIStatus(state, REGISTRATION_COMMANDS_DATA);
 
+export const selectApiDataErrorCode = state => {
```

Comment:
I'd expect this function returns a boolean, not an integer. Is that just my poor Javascript understanding?

Isn't this how you'd implement it quickest?
```javascript
export const selectApiDataErrorCode = state => {
  return selectAPIStatusData(state)?.response?.status;
}
```

Reply:
Yeah, I assume that if the status is not an error it wont be 403/other error code